src/client/components/componentWithSliders/allSliders/blockOfSliders/blockOfSliders.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './blockOfSliders.css';
+import OneSlider from '../slider/slider';
+
+const renderSliders = (numOfSliders) => {
+  const sliders = [];
+  for (let i = 0; i < numOfSliders; i++) {
+    sliders.push(<OneSlider key={i}/>);
+  }
+  return sliders;
+};
+
+const BlockOfSliders = (props) => (
```

Comment:
here all needed props could be districted like this const blocksliders = ({ name, numOfSliders}) = ()

Reply:
fixed